distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -922,14 +922,36 @@ type Column
     ## Parses a text column into values
 
        Arguments:
-       - type: Target type to parse the value to.
-         If `Auto`, will attempt to deduce the type automatically.
-       - format: For date/time types, the format to parse the value with.
-         For boolean types, two values that represent true and false separated
-         by `|`.
-       - on_problems: What to do if the value cannot be parsed.
-         By default, an `Invalid_Format` warning is attached and the value is
-         set to `Nothing`.
+       - type: The type to parse the column to. Defaults to `Auto` meaning that
+         the type will be inferred from the data. If all values fit one of the
+         more specific types, that type is selected. Otherwise the result
+         remains unchanged (a Text column). The types are tried in the following
+         order: Integer, Decimal, Date_Time, Date, Time_Of_Day, Boolean.
```

Comment:
I'd prefer the shorter form for this. Lets attach the details of what `Auto` does to itself.